packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -75,12 +82,19 @@ export const contract = async (
     },
   };
 
-  const makePortfolioKit = preparePortfolioKit(zone, { zcf });
+  const makePortfolioKit = preparePortfolioKit(zone, {
+    zcf,
+    axelarChainsMap,
+    chainHub,
```

Comment:
chainHub is excess authority; we don't want the portfolioKit to be able to _change_ what's in the chainHub.

see chainHubTools in #11505 (based on fast usdc).